src/components/Layout.svelte

```diff
@@ -35,8 +34,11 @@ $: if (visible) {
 
 onMount(() => { visible = true; });
 
+let width;
 </script>
 
+<svelte:window bind:innerWidth={width} />
```

Comment:
what is this for?

Reply:
I'm not honestly sure. It came from *GraphicBody*. 😄

Reply:
👍